packages/sdk/src/packageMetadata.ts

```diff
@@ -2,4 +2,4 @@
 // Licensed under the MIT license.
 
 export const name = "@microsoft/teamsfx";
-export const version = "0.1.2";
+export const version = "0.1.3";
```

Comment:
in the future, we also need to auto update the version here and run test case again when lerna bumps the version automatically.